web/crux/src/shared/grpc-node-connection.ts

```diff
@@ -97,6 +101,15 @@ export default class GrpcNodeConnection {
     this.statusChannel.complete()
     this.call.connection = null
   }
+
+  private getFirstItemOfStringArrayMetadata(key: string): string | null {
```

Comment:
Tests? 🤔

Reply:
it's a private method

Reply:
https://stackoverflow.com/a/56199483/5804254 ?